### PR TITLE
rg-159: Fix Registration Page Desktop Styles

### PR DESCRIPTION
**Pull request by Valentyn13**

### CODE REVIEW

**Comment by ValentinaTenus** [comment]
there is PR https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/pull/118 with updated styles

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx [hunk 1]
```diff
@@ -62,34 +62,48 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 className={styles.registration__form}
                 onSubmit={handleFormSubmit}
             >
-                <FormGroup error={errors.firstName} label="First Name">
+                <FormGroup
+                    className={styles.form_element_1}
```
Comment (MaxPoplavskiy) [inline frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx:66]: Consider changing to a more meaningful class name

**Review by MaxPoplavskiy — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx (hunk 1 above)
Comment (Valentyn13) [reply frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx]: Suggest something please :-)
grid_element_1, grid_element_2 etc. will be fine?

**Comment by Valentyn13** [comment]
> there is PR #118 with updated styles and it does not correspond to the layout given in the figma

I merged 188 pull request and fix conflicts, we had a bit different bugs on the same page. Now it's ok. Please check

**Review by MaxPoplavskiy — APPROVED** [review]

File: frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -63,7 +64,7 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
                 onSubmit={handleFormSubmit}
             >
                 <FormGroup
-                    className={styles.form_group}
+                    className={clsx(styles.form_group)}
```
Comment (MaxPoplavskiy) [inline frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx:67]: redundant clsx

**Review by MaxPoplavskiy — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/auth/components/social-media-links/social-media-links.tsx [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
+type Properties = {
+    className?: string;
+};
+
+const SocialMediaLinks: React.FC<Properties> = ({ className = '' }) => {
     return (
-        <div className={styles.social_media}>
+        <div className={clsx(styles.social_media, className)}>
             <span className={styles.social_media__description}>
                 Or log in with
```
Comment (jurok3x) [inline frontend/src/bundles/auth/components/social-media-links/social-media-links.tsx:15]: Should *log in*  be uppercase?

Comment (Valentyn13) [reply frontend/src/bundles/auth/components/social-media-links/social-media-links.tsx:15]: must be lowercase

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by ValentinaTenus — APPROVED** [review]

**Review by MaxPoplavskiy — APPROVED** [review]

**Review by siefimov — APPROVED** [review]

**Comment by Valentyn13** [comment]
[Registration page BR-15](https://trello.com/c/kqw4HlS3/159-registration-page-br-15)

**Pull request merged**